google/cloud/internal/log_wrapper_test.cc

```diff
@@ -60,6 +60,7 @@ TEST(LogWrapper, DefaultOptions) {
   TracingOptions tracing_options;
   // clang-format off
   std::string const text =
+      R"pb(google.spanner.v1.Mutation { )pb"
```

Comment:
Feels like this could be a single raw string, maybe for another PR.

Reply:
I think the lack of newlines dooms this to being some form of concatenation.